spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -153,9 +216,9 @@ def main(argv=None):
     if arguments.o is not None:
         dest_folder = arguments.o
     else:
-        dest_folder = os.path.join(os.path.abspath(os.curdir), data_name)
+        dest_folder = f"{DATASET_DICT[data_name]['default_location']}/{data_name}"
```

Comment:
My thinking was that hardcoding the `data_name` might be a little redundant and a little more error prone? Since we're duplicating information, and all of the folder names end up matching the dataset name anyway.

But, on reflection, I think I like your idea more! It keeps things more readable, since you only have to look in one place to find out how the path for a dataset gets set.

Reply:
In fact, your suggestion revealed a bug in my changes. The `binaries_linux` and `binaries_osx` downloads _don't_ use a subfolder with the same name. So, my initial assumption was wrong, and your proposed changes are necessary. :)